test/README.md

```diff
@@ -33,36 +37,47 @@ these steps:
   2. Copy and/or rename the `tmp/server/debug/rippled.cfg` file to somewhere
   convenient.
 
-4. Start `rippled` (in a debugger) with command line options
+##### Using the command line
+
+1. Create a `rippled.cfg` file for the tests.
+  1. Run `npm test --noserver`. The tests will fail. **This failure is expected.**
```

Comment:
It's a near duplicate of the last step of the previous section using config
files.
On Jan 26, 2015 8:09 PM, "Tom Swirly" <notifications@github.com> wrote:

> In test/README.md
> <https://github.com/ripple/rippled/pull/831#discussion_r23579785>:
>
> > @@ -33,36 +37,47 @@ these steps:
> >    2. Copy and/or rename the `tmp/server/debug/rippled.cfg` file to somewhere
> >    convenient.
> >
> > -4. Start `rippled` (in a debugger) with command line options
> > +##### Using the command line
> > +
> > +1. Create a `rippled.cfg` file for the tests.
> > +  1. Run `npm test --noserver`. The tests will fail. **This failure is expected.**
>
> I thought I saw This failure is expected. before?! But it seems to be new
> in this change? Baffling!
>
> Thanks for adding documentation!!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/rippled/pull/831/files#r23579785>.
>